day_7/high_level.md

```diff
@@ -0,0 +1,53 @@
+#Caesar Cipher High-Level Notes
+
+##Big Steps
```

Comment:
Great idea starting with the big steps and getting more and more detailed as you think through the problem.